CONTRIBUTING.md

```diff
@@ -8,49 +8,39 @@ to the rules described here, but by following the instructions below you
 should have a much easier time getting your work merged with the upstream
 project.
 
-* Test Your Code
+## Test Your Code Using Existing Tests
 
-There are two possible tests you can run to verify your code.  The first test
-is used to check the formatting and coding style of your changes, you can run
-the test with the following command:
+There are three possible tests you can run to verify your code.  The first
+test is used to check the formatting and coding style of your changes, you
+can run the test with the following command:
 
 	# make check-syntax
 
 ... if there are any problems with your changes a diff/patch will be shown
 which indicates the problems and how to fix them.
 
-The second possible test is used to ensure the sanity of your code changes
-and to test these changes against the included tests.  You can run the test
-with the following command:
+The second possible test is used to ensure that the different internal syscall
+tables are consistent and to test your changes against the automated test
+suite.  You can run the test with the following command:
```

Comment:
I think the text needs some tweaking, especially the mention of the syscall tables (the golang bindings don't have any).  Perhaps go back to the original text here?